Formula/php.rb

```diff
@@ -0,0 +1,412 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net"
+  url "https://github.com/php/php-src/archive/php-7.1.7.tar.gz"
+  sha256 "fc57fde1df31e34fc6d58ea4ec477429d3663187391e2b307444b532dc18550d"
+
+  # javian: not sure about the origin of this so I'll keep it commented for now. Could it have something to do with building extensions ?
+  # So PHP extensions don't report missing symbols
+  #skip_clean "bin", "sbin"
+
+  # TODO
+  # Opcache default config file is included in extension formula, is it needed ?
+  # ldap extension: I have a vague recollection that this caused some issues that were resolved with exluding it. needs to be checked.
+  # According to php docs enchant requires aspell as a dep, should not be an issue since aspell is required.
+  # Need to add tests for all supported SAPIs
+  # Should we remove --with-pdo-dblib from formula ? (mssql support is long gone and as far as I can tell this is the 
+  #  only purpose and would also exclude a dep, freetds)
+  # How should the Formula handle the apache module ? Sierra (have not checked high sierra) can't support building it for the OS 
+  #   (without manually fiddling with link in the file system) with the bundles tools
+
+  option "with-debug-symbols", "Compile with debugging symbols"
+  option "with-freetds", "Build MSSQL-DB support extension"
+  option "with-httpd24", "Enable building of shared Apache 2.4 Handler module"
+  option "with-httpd22", "Enable building of shared Apache 2.2 Handler module"
+  option "with-imap-uw", "Build IMAP extension"
+  option "with-thread-safety", "Build with thread safety"
+
+  # javian: Not yet checked these options
+  option "with-pdo-oci", "Include Oracle databases (requries ORACLE_HOME be set)"
+  option "without-ldap", "Build without LDAP support"
+  option "without-unixodbc", "Build without unixODBC support"
+
+  depends_on "autoconf" => :build
+  depends_on "bison" => :build
+  depends_on "flex" => :build
+  depends_on "re2c" => :build
+  depends_on "enchant" => :optional
+  depends_on "homebrew/apache/httpd24" => :optional
```

Comment:
Since the httpd module can be used with apache that comes with macOS do you think we should make it a :build requirement ? I'm not sure what the most common use case is but in the php tap there's been several requests to make it work with the stock apache. Currently you can compile the module and remove the httpd24 formula and it will still work but it wouldn't really be a scenario that would be supported.